cmd/genman/gen_kube_man.go

```diff
@@ -39,6 +41,7 @@ import (
 )
 
 func main() {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Ditto here.

Reply:
I think this is in the same situation as the docs generator (this generates the man pages)

Reply:
Removed.